content/references/protocol-reference/transactions/transaction-types/accountset.md

```diff
@@ -35,12 +35,12 @@ An AccountSet transaction modifies the properties of an [account in the XRP Ledg
 |:-----------------|:-----------------|:------------------|:-------------------|
 | [`ClearFlag`](#accountset-flags) | Number | UInt32      | _(Optional)_ Unique identifier of a flag to disable for this account. |
 | [`Domain`](#domain) | String        | Blob              | _(Optional)_ The domain that owns this account, as a string of hex representing the ASCII for the domain in lowercase. [Cannot be more than 256 bytes in length.](https://github.com/ripple/rippled/blob/55dc7a252e08a0b02cd5aa39e9b4777af3eafe77/src/ripple/app/tx/impl/SetAccount.h#L34) |
-| `EmailHash`      | String           | Hash128           | _(Optional)_ Hash of an email address to be used for generating an avatar image. Conventionally, clients use [Gravatar](http://en.gravatar.com/site/implement/hash/) to display this image. |
+| `EmailHash`      | String           | Hash128           | _(Optional)_ An arbitrary 128-bit value. Conventionally, clients treat this as the md5 hash of an email address to use for displaying a [Gravatar](http://en.gravatar.com/site/implement/hash/) image. |
```

Comment:
Incidental, but updated for consistency with the description of `WalletLocator`. The rewritten description makes it clearer that the protocol does not actually care if this is a real md5 hash.